CIFAR/data.swift

```diff
@@ -13,12 +15,12 @@ func maybeDownload(to directory: String = ".") {
     }
 }
 
-func loadBatches(from file: String, in directory: String = ".")
-    -> (Tensor<Int32>, Tensor<Float>) {
-    maybeDownload(to: directory)
+// Each CIFAR data file is provided as a Python pickle of NumPy arrays
+func loadCIFARFile(filename: String, in directory: String = ".") -> (Tensor<Int32>, Tensor<Float>) {
```

Comment:
According to the Swift API Design Guidelines, if it's a normal function (not an initializer or factory method), then the first argument label should form a phrase with the function base name when possible.

```suggestion
func loadCIFARFile(named name: String, in directory: String = ".") -> (Tensor<Int32>, Tensor<Float>) {
```